Automattic/vip-go-mu-plugins
# Rollout Enterprise Search query cache to 50% of sites + all non-prods

## Description

We've been returning the fallback values on timeouts for quite some time now, it's time to roll it out for all requests.

## Changelog Description

### Plugin Updated: Enterprise Search

Roughly 50% of production sites and all non-production environments now utilize cached queries to avoid extra requests to Enterprise Search servers. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

This PR shouldn't have any change for local/non-prods. 
1. Search for something, verify the remote request is made via Query Monitor/Debug Bar
2. Refresh the page and verify that no remote request is happening.
